pkg/ccloudv2/kafkarest.go

```diff
@@ -341,3 +366,29 @@ func (c *KafkaRestClient) UpdateKafkaTopicPartitionCount(topicName string, updat
 func (c *KafkaRestClient) GetKafkaTopic(topicName string) (kafkarestv3.TopicData, *http.Response, error) {
 	return c.TopicV3Api.GetKafkaTopic(c.kafkaRestApiContext(), c.ClusterId, topicName).Execute()
 }
+
+// GetShareGroupTopicNames extracts unique topic names from assigned topic partitions
+func (c *KafkaRestClient) GetShareGroupTopicNames(shareGroup kafkarestv3.ShareGroupData) string {
```

Comment:
Let's move this function out of the kafkarest ccloudv2 file and into the `command_share_group.go` file since we want the ccloudv2 pkg functions to just be wrappers for the SDK calls, and not do additional processing